qoqo-qryd/src/pragma_operations.rs

```diff
@@ -104,8 +104,9 @@ impl PragmaChangeQRydLayoutWrapper {
     /// Returns:
     ///     set[int]: The involved qubits of the PRAGMA operation.
     fn involved_qubits(&self) -> PyObject {
-        let pyobject: PyObject =
-            Python::with_gil(|py| -> PyObject { PySet::new(py, &["All"]).unwrap().to_object(py) });
+        let pyobject: PyObject = Python::with_gil(|py| -> PyObject {
+            PySet::new_bound(py, &["All"]).unwrap().to_object(py)
+        });
```

Comment:
This is old stuff, but now that I think about it: is there a way to make a one-liner of this? Nothing to do with 0.21, I guess it was possible in the old code too.